File: javascript/bee1020.js
const input = require("fs").readFileSync("stdin", "utf-8");

const lines = input.split(" ")

let valor = lines.shift()

const anos = parseInt(valor / 365)
valor = valor % 365

const meses = parseInt(valor / 30)
valor = valor % 30

const dias = valor

console.log(`${anos} ano(s)`)
console.log(`${meses} mes(es)`)
console.log(`${dias} dia(s)`)

File: python/bee1020.py
# -*- coding: utf-8 -*-

entrada = int((input()))
resto = 0

a = int(entrada / 365)
resto = entrada % 365

m = int(resto / 30)
resto = resto % 30

print(f"{a} ano(s)\n{m} mes(es)\n{resto} dia(s)")